tests/integration_tests/saas/test_stripe_task.py

```diff
@@ -1071,9 +1071,8 @@ def test_stripe_erasure_request_task(
         ],
     )
 
-    # run erasure with MASKING_STRICT to execute the update actions
-
-    config.execution.MASKING_STRICT = True
+    # Run erasure with masking_strict = False so both update and delete actions can be used
+    config.execution.MASKING_STRICT = False
```

Comment:
Ideally we wouldn't set this in the test since if execution halts mid-test the value won't be reset and could cause cascade failures within other tests. Looks like you're only updating the value here so let's change these as part of a [subsequent ticket](https://github.com/ethyca/fidesops/issues/606).